src/NumericalAlgorithms/SpatialDiscretization/OptionTags.hpp

```diff
@@ -0,0 +1,20 @@
+// // Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "Options/Options.hpp"
+
+namespace SpatialDiscretization::OptionTags {
+/*!
+ * \brief Group holding all the options for the spatial discretization of the
```

Comment:
I certainly agree with your statement. Probably the right thing to do in the long term is to move `Domain` into this group since, as you point out, the domain is an essential part of the spatial discretization.

I'll expand the documentation until we've decided to move the domain into this group. Other related input file changes I can see:
- limiters are part of the spatial discretization, I think
- the initial data should probably be in the `EvolutionSystem` group

Thoughts on those ideas? (for a separate PR)

Reply:
Yeah, I like those ideas for future option refactoring